File: C#/Server/DataTipes/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataTypes
{
    public class Message
    {
        public long ID { get; set; }
        public string text { get; set; }
        public int idTo { get; set; }
        public int idFrom { get; set; }
        public DateTime date { get; set; }

        public Message()
        {
            text = "";
            idTo = 0;
            idFrom = 0;
            date = DateTime.Now;
        }
    }
}

File: Node,Mongoos/DataTypes/Message.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var MessageSchema = new Schema({
    text: String,
    idTo: String,
    idFrom: String,
    date: {type: Date, default: Date.now },
})

var Message = mongoose.model('Messages', MessageSchema)
model.exports = Message;